java/ql/src/Violations of Best Practice/Magic Constants/MagicConstants.qll

```diff
@@ -401,7 +391,7 @@ predicate literalInsteadOfConstant(
   exists(string context |
     canUseFieldInsteadOfLiteral(constField, magicLiteral, context) and
     message =
-      "Literal value '" + magicLiteral.getLiteral() + "' used " + " in a call to " + context +
+      "Literal value '" + magicLiteral.getValue() + "' used " + " in a call to " + context +
```

Comment:
In a human-consumable message we probably should use the representation used in the source

Reply:
Makes sense, have reverted this in the latest commit.